src/components/payment/staff-payment.vue

```diff
@@ -36,12 +36,27 @@ export default class StaffPayment extends Vue {
   @Action setStaffPayment!: ActionBindingIF
 
   /** Whether to validate the data. */
-  @Prop({ default: false }) readonly doValidate: boolean
+  private validate = false
+
+  /** Whether to show the staff payment invalid section styling. */
+  private get showStaffPaymentInvalidSection (): boolean {
+    const option = this.getStaffPayment.option ?? StaffPaymentOptions.NONE
+    // True if no option is selected
+    return this.validate && (option === StaffPaymentOptions.NONE)
+  }
+
+  /** Called externally to set validation state. */
+  public setValidation (val: boolean) {
+    this.validate = val
```

Comment:
This is now a state, so I can toggle it True or False as needed (instead of making it responsive like a prop). I need this state to "stick" so that I can enable it when the user tries to submit this NR, and clear it under various scenarios (like reloading this tab, selecting a different radio button, etc). I was not able to do that without a state.